package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package format="2">
+<package format="3">
```

Comment:
Is this going to work on older systems ?
It's not even on the wiki yet: http://wiki.ros.org/catkin/package.xml

Reply:
It's supported since `rosdep` version 0.15.0: https://github.com/ros-infrastructure/rosdep/blob/master/CHANGELOG.rst#0150-2019-01-24

Do you see a system that relies on an older `rosdep` version?

It's been finalised in REP0149: https://www.ros.org/reps/rep-0149.html#package-format-3
And the XSD is here: http://download.ros.org/schema/package_format3.xsd

Reply:
ROS Kinetic / Ubuntu Xenial has 0.18.0 and I believe that is the oldest setting are supporting.

Reply:
ok, thanks :)